sprocs/question_order.sql

```diff
@@ -26,7 +26,7 @@ WITH locks_next AS (
         JOIN instance_questions AS iq ON (iq.assessment_instance_id = ai.id)
         JOIN assessment_questions AS aq ON (aq.id = iq.assessment_question_id)
     WHERE
-        ai.id = assessment_instance_id
+        ai.id = arg_assessment_instance_id
```

Comment:
Without the `arg_` this was being interpreted by postgres as `ai.id = iq.assessment_instance_id`, so it was searching over all assessment instances in the DB, not just the one we wanted.